packages/lib-classifier/src/plugins/tasks/survey/components/SurveyTaskContainer.js

```diff
@@ -134,10 +147,12 @@ class SurveyTaskContainer extends React.Component {
         answers={answers}
         disabled={disabled}
         filters={filters}
+        focusedChoice={focusedChoice}
         handleAnswers={this.handleAnswers.bind(this)}
         handleChoice={this.handleChoice.bind(this)}
         handleDelete={this.handleDelete.bind(this)}
         handleFilter={this.handleFilter.bind(this)}
+        handleFocusedChoice={this.handleFocusedChoice.bind(this)}
```

Comment:
I missed this before, but you shouldn’t run `.bind` on every render. It creates a new, bound function each time it runs. You should bind your event handlers once, in the component constructor, like this:

```js
this.handleEvent = this.handleEvent.bind(this)
```

You can then safely reference `this.handleEvent` in your component.

All the event handlers here need to be updated.